Client/Frontend/Browser/TopTabsViews.swift

```diff
@@ -31,19 +31,9 @@ class TopTabsHeaderFooter: UICollectionReusableView {
 
     func arrangeLine(_ kind: String) {
         line.snp.removeConstraints()
-        switch kind {
-            case UICollectionElementKindSectionHeader:
```

Comment:
I couldn't see why this header/footer logic is needed, I can put it back if it is.